src/hct_mis_api/middlewares/xframe.py

```diff
@@ -0,0 +1,25 @@
+from typing import List, Optional
+
+from django.http import HttpRequest, HttpResponse
+from django.utils.deprecation import MiddlewareMixin
```

Comment:
@johniak, @MarekBiczysko  Please check if this middleware is correct. I needed to overwrite the default django setting in order to display dashboard page in an iframe